README.md

```diff
@@ -1,169 +1,129 @@
-| `ingress.hosts`                              | List of domain names Kubernetes Ingress rule looks for. Set it to the domains in which Atlantis will be hosted on.                                                                                                                                                                                                           | `chart-example.local`     |
-| `ingress.hosts[0].paths`                     | List of paths to use in Kubernetes Ingress rules.  Should be set to `/*` if using gce-ingress in Google                                                                                                                                                                                                        | `[/]`     |
-| `ingress.tls`                               | Kubernetes tls block. See [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls) for details.                                                                                                                                                                            | `[]`     |
-| `test.enabled`                              | Whether to enable the test. | `true` |
-| `extraManifests`                         | add additional manifests to deploy                      | `[]`                      |
-| `initContainers`                            | Containers used to initialize context for Atlantis pods                                  | `[]`                              |
-
-**NOTE**: All the [Server Configurations](https://www.runatlantis.io/docs/server-configuration.html) are passed as [Environment Variables](https://www.runatlantis.io/docs/server-configuration.html#environment-variables).
-
-
-## Upgrading
-### From `2.*` to `3.*`
-* The following value names have been removed. They are replaced by [Server Side Repo Configuration](https://www.runatlantis.io/docs/server-side-repo-config.html)
-  * `requireApproval`
-  * `requireMergeable`
-  * `allowRepoConfig`
-
-To replicate your previous configuration, run Atlantis locally with your previous flags and Atlantis will print out the equivalent repo-config, for example:
-
-```bash
-$ atlantis server --allow-repo-config --require-approval --require-mergeable --gh-user=foo --gh-token=bar --repo-whitelist='*'
-WARNING: Flags --require-approval, --require-mergeable and --allow-repo-config have been deprecated.
-Create a --repo-config file with the following config instead:
-
----
-repos:
-- id: /.*/
-  apply_requirements: [approved, mergeable]
-  allowed_overrides: [apply_requirements, workflow]
-  allow_custom_workflows: true
-
-or use --repo-config-json='{"repos":[{"id":"/.*/", "apply_requirements":["approved", "mergeable"], "allowed_overrides":["apply_requirements","workflow"], "allow_custom_workflows":true}]}'
 ```
+usage: procore_terraform_wrapper.py [-h] --action {init,plan,apply}
```

Comment:
```suggestion
usage: ./procore_terraform_wrapper.py [-h] --action {init,plan,apply}
```

Reply:
I generated this by copy-pasting the output of `./procore_terraform_wrapper.py -h`. The name of the script is automatically generated by argparse. I'd rather not change it manually in the readme.

Reply:
I would prefer to make the readme the source of copy/pasteable commands. I think it is easier when someone is completely unfamiliar with the repo. But that doesn't need to be a blocker if we disagree